hbase-server/src/test/java/org/apache/hadoop/hbase/mob/MobStressToolRunner.java

```diff
@@ -0,0 +1,325 @@
+
+ 1. Settings: Region Size 200 MB,  Flush threshold 800 KB.
+ 2. Insert 10 Million records
+ 3. MOB Compaction and Archiver
+      a) Trigger MOB Compaction (every 2 minutes)
+      b) Trigger major compaction (every 2 minutes)
+      c) Trigger archive cleaner (every 3 minutes)
+ 4. Validate MOB data after complete data load.
+
+ This class is used by MobStressTool only. This is not a unit test
+
+ */
+@SuppressWarnings("deprecation")
+public class MobStressToolRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(MobStressToolRunner.class);
+
+
+  private HBaseTestingUtility HTU;
+
+  private final static String famStr = "f1";
+  private final static byte[] fam = Bytes.toBytes(famStr);
+  private final static byte[] qualifier = Bytes.toBytes("q1");
+  private final static long mobLen = 10;
+  private final static byte[] mobVal = Bytes
+      .toBytes("01234567890123456789012345678901234567890123456789012345678901234567890123456789");
+
+  private Configuration conf;
+  private HTableDescriptor hdt;
+  private HColumnDescriptor hcd;
+  private Admin admin;
+  private long count = 500000;
+  private double failureProb = 0.1;
+  private Table table = null;
+  private MobFileCleanerChore chore = new MobFileCleanerChore();
+
+  private static volatile boolean run = true;
+
+  public MobStressToolRunner() {
+
+  }
+
+  public void init(Configuration conf, long numRows) throws IOException {
+    this.conf = conf;
+    this.count = numRows;
+    initConf();
+    printConf();
+    hdt = createTableDescriptor("testMobCompactTable");
+    Connection conn = ConnectionFactory.createConnection(this.conf);
+    this.admin = conn.getAdmin();
+    this.hcd = new HColumnDescriptor(fam);
+    this.hcd.setMobEnabled(true);
+    this.hcd.setMobThreshold(mobLen);
+    this.hcd.setMaxVersions(1);
+    this.hdt.addFamily(hcd);
+    if (admin.tableExists(hdt.getTableName())) {
+      admin.disableTable(hdt.getTableName());
+      admin.deleteTable(hdt.getTableName());
+    }
+    admin.createTable(hdt);
+    table = conn.getTable(hdt.getTableName());
+  }
+
+  private void printConf() {
+    LOG.info("Please ensure the following HBase configuration is set:");
+    LOG.info("hfile.format.version=3");
+    LOG.info("hbase.master.hfilecleaner.ttl=0");
+    LOG.info("hbase.hregion.max.filesize=200000000");
+    LOG.info("hbase.client.retries.number=100");
+    LOG.info("hbase.hregion.memstore.flush.size=800000");
+    LOG.info("hbase.hstore.blockingStoreFiles=150");
+    LOG.info("hbase.hstore.compaction.throughput.lower.bound=50000000");
+    LOG.info("hbase.hstore.compaction.throughput.higher.bound=100000000");
+    LOG.info("hbase.master.mob.cleaner.period=0");
+    LOG.info("hbase.mob.default.compactor=org.apache.hadoop.hbase.mob.FaultyMobStoreCompactor");
+    LOG.warn("hbase.mob.compaction.fault.probability=x, where x is between 0. and 1.");
+
+  }
+
+  private HTableDescriptor createTableDescriptor(final String name, final int minVersions,
```

Comment:
why do we have two versions of this method? It looks like we don't actually use any of these options. Can we just put the `new HTableDescriptor` call directly in the init block?